src/main/scala/hmda/model/filing/ts/TransmittalSheet.scala

```diff
@@ -0,0 +1,24 @@
+package hmda.model.filing.ts
+
+import hmda.model.filing.HmdaFileRow
+import hmda.model.institution.{Agency, UndeterminedAgency}
+
+case class TransmittalSheet(
+    id: Int = 1,
+    institutionName: String = "",
+    year: Int = 2018,
+    quarter: Int = 4,
```

Comment:
Why does the quarter default to 4 here?  Is the normal range 0-3?

Reply:
[Filing Instruction Guide](https://www.consumerfinance.gov/data-research/hmda/static/for-filers/2018/2018-HMDA-FIG.pdf), page 20: Calendar Quarter Valid Values: 4